spec/helpers/override_helper_spec.rb

```diff
@@ -21,15 +21,27 @@
 
   describe '#complex_reason_label' do
     context 'when a prison POM' do
-      let(:subject) { OffenderPresenter.new(OpenStruct.new(immigration_case?: true), nil) }
+      # we need to set up this test to return a Prison POM recommendation; we are using an
```

Comment:
Is there a way to express this information through the context blocks rather than comments? Comments are more likely to rot, I think.

Reply:
we could have this, 
![image](https://user-images.githubusercontent.com/46002877/73650266-a74c3d00-4679-11ea-9da9-56df857d04db.png)
however were losing some of the context that immigration cases always evaluate to prison pom. not sure how important this information is

Reply:
I like that better! I'm not sure what the `context "when a prison POM"` adds by itself as well. I'm guessing the other branch of this (on line 33, "when a probation POM") is setting up an offender that's expected to be recommended to a probation POM?

You have more context than I do, but the ideal structure to me looks like:
```ruby
describe '#complex_reason_label' do
  context 'with an immigration case, which requires a prison POM' do
    # SNIP: stuff setting up the immigration case
    it 'notes the prisoner is not appropriate for a probation POM' do
      expect(subject.complex_reason_label).to eq('Prisoner assessed as not suitable for a prison officer POM')
    end
  end

  context 'with a case that is suitable for a probation POM' do
    # SNIP: stuff setting up that other offender

    it "notes that the prisoner can be assigned to a probation POM" do
      expect(subject.complex_reason_label).to eq('Prisoner assessed as suitable for a prison officer POM despite tiering calculation')
    end
  end
end
```